templates/containers/what-are-containers.html

```diff
@@ -14,282 +14,250 @@
+        <h2>
+          Containers are a virtualisation technology that isolates each process in a lightweight and standard environment.
+        </h2>
+        <p class="p-strip is-shallow">
+          They let users package and quickly deploy entire applications parallel to each other on the same kernel and hardware, while maintaining isolation among the workloads.
         </p>
+        <hr class="is-muted" />
       </div>
-      <div class="col-5 u-align--center u-vertically-center u-hide--medium u-hide--small">
-        {{ image(url="https://assets.ubuntu.com/v1/fece8a11-container-in-the-cloud.svg",
-                alt="",
-                width="200",
-                height="163",
-                hi_def=True,
-                loading="auto") | safe
-        }}
+      <div class="p-media-container u-darker-background col-3 u-align--center u-vertically-center u-hide--small u-hide--medium">
+        <img src="https://assets.ubuntu.com/v1/0b0cef3c-hero.png"
+             alt="" />
       </div>
     </div>
   </section>
 
-  <section class="p-strip--light">
-    <div class="u-fixed-width">
-      <h2>Container virtualisation</h2>
-      <p>
-        Containers are instantiated from container images, a layered packaging system, to distribute applications along with their configuration and runtime dependencies.
-      </p>
-      <p>
-        From development to production, containers power modern cloud applications as they allow developers to move fast between platforms and versions.
-      </p>
-      <p>
-        Inside Google alone, at least <a href="https://cloud.google.com/containers/">two billion containers are generated each week</a> to manage its vast operations. There are many tools to create, deploy and manage containers, including LXD, Docker and Kubernetes.
-      </p>
-    </div>
-  </section>
-
-  <section class="p-strip">
-    <div class="u-fixed-width">
-      <h2>What is Docker?</h2>
-      <p>
-        <a href="https://www.docker.com/">Docker</a>  is an open-source containerisation platform. It provides an easy way to build and deploy containers in the cloud or on-premises. Docker didn’t introduce container technologies, but they made the user experience of container technologies so easy for developers that Docker soon became shorthand for containers and the most popular container format. <a href="https://www.docker.com/">Docker Inc</a>, the company that supports the Docker project and community, also operates <a href="https://hub.docker.com/">Docker Hub</a>, a space for hosting container images.
-      </p>
+  <section class="p-section u-fixed-width">
+    <hr class="p-rule" />
+    <div class="row--50-50 p-section--shallow">
+      <div class="col">
```

Comment:
```suggestion
  <section class="p-section">
    <div class="row--50-50">
        <hr class="p-rule" />
      <div class="col">
```
`u-fixed-width` is unnecessary here as it's the the same thing as a row. We just need to put the separator within it to align correctly

Reply:
Please apply the same for the relevant sections throughout the file